src/js/main.js

```
@@ -0,0 +1,621 @@
+        document.getElementById('disable-lift-btn').addEventListener('click', () => {
+            this.disableLift();
+        });        document.getElementById('enable-all-lifts-btn').addEventListener('click', () => {
+            this.enableAllLifts();
+        });
+    }
+
+    generateSimulation() {
+        const floorsInput = document.getElementById('floors');
+        const liftsInput = document.getElementById('lifts');
+        
+        this.floors = parseInt(floorsInput.value);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)